src/test/java/no/regnskap/integration/RegnskapApiIntegration.java

```diff
@@ -0,0 +1,83 @@
+package no.regnskap.integration;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import no.regnskap.model.RegnskapDB;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.codecs.pojo.PojoCodecProvider;
+import org.junit.*;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static no.regnskap.TestData.*;
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RegnskapApiIntegration {
+    private static String regnskapRootURL;
+
+    @ClassRule
+    public static DockerComposeContainer compose = new DockerComposeContainer<>(new File("src/test/resources/test-compose.yml"))
```

Comment:
Specifying file like this is wrong on so many levels, but it seems it is the only option  :-/

Reply:
It's how they do it in the documentation.

https://www.testcontainers.org/modules/docker_compose/

Reply:
Yes, I checked the documentation and the source, and you have to specify a filename, but they should really have a stream-version of their constructor - the src file will not exist if you run the test from .jar

Reply:
I made an issue where I paraphrased your comment

https://github.com/testcontainers/testcontainers-java/issues/1506